File: 0167-two-sum-ii-input-array-is-sorted/0167-two-sum-ii-input-array-is-sorted.js
/**
 * @param {number[]} nums
 * @param {number} target
 * @return {number[]}
 */
var twoSum = function(nums, target) {
    let Lptr = 0;
    let Rptr = nums.length -1;
    

    for(let i = 0 ; i<nums.length ; i++){
        
       if(nums[Lptr] + nums[Rptr] == target){
        return [Lptr+1,Rptr+1]
       }if(nums[Lptr] + nums[Rptr] > target){
        Rptr--;
       }if(nums[Lptr] + nums[Rptr] < target){
        Lptr++;
       }

    }
    
};

File: 0167-two-sum-ii-input-array-is-sorted/0167-two-sum-ii-input-array-is-sorted.py
class Solution:
    def twoSum(self, numbers: List[int], target: int) -> List[int]:
        Lptr = 0
        Rptr = len(numbers) - 1

        # for i in range(len(numbers)):
        while Lptr < Rptr:
            # print(i)

            if(numbers[Lptr] + numbers[Rptr] == target):
                return [Lptr + 1 , Rptr + 1]
            if(numbers[Lptr] + numbers[Rptr] > target):
                Rptr -=1
            if(numbers[Lptr] + numbers[Rptr] < target):
                Lptr +=1
            

        